lib/cc/engine/analyzers/violation.rb

```diff
@@ -32,12 +39,24 @@ def report_name
           "#{current_sexp.file}-#{current_sexp.line}"
         end
 
+        def mass # ABH Reverse engineer the actual mass from flay score.
```

Comment:
Okay, I can see if the Sexp has the mass/node_size attached. If so, that would be great.